app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -94,10 +105,33 @@ class StateFragmentPresenter @Inject constructor(
 
   private fun subscribeToCurrentState() {
     ephemeralStateLiveData.observe(fragment, Observer<EphemeralState> { result ->
+      try {
+        if (result.state.interaction.id.equals("NumericInput"))
+          addNumberInputContentCard(result.state.interaction.customizationArgsMap.get("placeholder")!!.normalizedString)
+      } catch (e: Exception) {
+      }
```

Comment:
Maybe add logger in this exception.

Reply:
try {} catch is removed as per the verbal conversation